src/robusta/core/sinks/msteams/msteams_sink.py

```diff
@@ -8,8 +8,9 @@ class MsTeamsSink(SinkBase):
     def __init__(self, sink_config: MsTeamsSinkConfigWrapper, registry):
         super().__init__(sink_config.ms_teams_sink, registry)
         self.webhook_url = sink_config.ms_teams_sink.webhook_url
+        self.use_emoji = False if (sink_config.ms_teams_sink.use_emoji is None) else sink_config.ms_teams_sink.use_emoji
```

Comment:
This could be simplified to:
`self.use_emoji = sink_config.ms_teams_sink.use_emoji`
if the sink parameter has default value